src/Components/Header/Header.js

```diff
@@ -6,10 +6,6 @@ const Header = () => {
     <nav className='header'>
       <h1>Tainted Tomatillos</h1>
       <h2 className='small-title'>Where The Ratings Are Honestly Rotten</h2>
-      <button className='search-button'>Search</button>
```

Comment:
Possibly rendering the <SearchBar /> in here could help with the debugging of the input line.  Although I believe that solution lives in the app component.  I will take a peak today.